snapd.go

```diff
@@ -422,7 +457,6 @@ func action(ctx *cli.Context) {
 	if autodiscoverPath != "" {
 		log.Info("auto discover path is enabled")
 		paths := filepath.SplitList(autodiscoverPath)
-		c.SetAutodiscoverPaths(paths)
```

Comment:
Is there a reason AutoDiscoverPaths is removed out of Control struct? Granted it does nothing right now, but auto discovering plugins (and eventually monitoring for plugin in said directories) needs to move out of snapd and into Control. Unless this is directly related to RPC changes, I would not remove this and let any changes regarding this be handled by a PR relating to plugin discovery changes.